Unix/provreg/provreg.c

```diff
@@ -511,7 +511,7 @@ static int _AddEntry(
 #endif /* defined(CONFIG_ENABLE_PREEXEC) */
 
     /* ProvRegEntry.hosting*/
-    e->hosting = PROV_HOSTING_INPROC;
+    e->hosting = PROV_HOSTING_REQUESTOR;
```

Comment:
So is this changing the default hosting type to `@requestor@`, assuming that the `HOSTING=` line isn't specified?

Where is that parsed from the .reg file? Is `inproc` disabled now? Or will it be a separate commit?

If a separate commit, what's the purpose of the changes here?